traits/editor_factories.py

```diff
@@ -141,6 +151,30 @@ def date_editor():
     return DateEditor
 
 
+def datetime_editor():
+    """ Factory function that returns an editor with date & time for
+    editing DateTime values.
+    """
+
+    try:
+        from traitsui.api import DateTimeEditor
+
+        return DateTimeEditor()
+
+    except ImportError:
+
+        logger.warn(msg="Could not import DateTimeEditor from "
+                        "traitsui.api, using TextEditor instead")
+
+        from traitsui.api import TextEditor
+
+        iso_frmt = "%Y-%m-%dT%H:%M:%S"
+        return TextEditor(
+            evaluate=lambda dtstr: datetime.datetime.strptime(dtstr, iso_frmt),
```

Comment:
Nit: I think just make some helper functions for these at the module level.  That also potentially allows re-use.

Reply:
Yes, made the changes. Thanks!